app/styles/_mixins.less

```diff
@@ -94,7 +94,7 @@
   margin-right: 10px;
 }
 
-// additional visiblility utility for xlg 
+// additional visiblility utility for xlg
```

Comment:
I think the use of `!important` here makes sense, given what its doing.  @sg00dwin @rhamilto feel free to affirm/refute :smile: